python/paddle/fluid/contrib/slim/tests/CMakeLists.txt

```diff
@@ -98,10 +104,10 @@ list(REMOVE_ITEM TEST_OPS test_mkldnn_int8_quantization_strategy)
 # QAT FP32 & INT8 comparison python api tests
 if(LINUX AND WITH_MKLDNN)
 	set(DATASET_DIR "${INFERENCE_DEMO_INSTALL_DIR}/int8v2")
-	set(QAT_DATA_DIR "${INFERENCE_DEMO_INSTALL_DIR}/int8v2")
-	set(QAT_MODELS_BASE_URL "${INFERENCE_URL}/int8/QAT_models")
-	set(MKLDNN_QAT_TEST_FILE "qat_int8_comparison.py")
-	set(MKLDNN_QAT_TEST_FILE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/${MKLDNN_QAT_TEST_FILE}")
+      set(QAT_DATA_DIR "${INFERENCE_DEMO_INSTALL_DIR}/int8v2")
+      set(QAT_MODELS_BASE_URL "${INFERENCE_URL}/int8/QAT_models")
+      set(MKLDNN_QAT_TEST_FILE "qat_int8_comparison.py")
+      set(MKLDNN_QAT_TEST_FILE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/${MKLDNN_QAT_TEST_FILE}")
```

Comment:
@wojtuss Should I spend time to find how to remove this diff? It seems because different indent numbers of Tab

Reply:
it looks like you have put spaces instead of tabs here, please try using tabs